usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -108,7 +108,7 @@ fi
 # When DHCP is used via 58-start-dhclient.sh do not change the existing networking setup here:
 cat - <<EOT >>$network_devices_setup_script
 # When DHCP was used via 58-start-dhclient.sh do not change the existing networking setup here:
-[[ ! -z "\$USE_DHCLIENT" && -z "\$USE_STATIC_NETWORKING" ]] && return
+is_true \$USE_DHCLIENT && ( [[ -z "\$USE_STATIC_NETWORKING" ]] || is_false \$USE_STATIC_NETWORKING ) && return
```

Comment:
I think using `! is_true $USE_STATIC_NETWORKING`
makes the code simpler as in
```
is_true \$USE_DHCLIENT && ! is_true \$USE_STATIC_NETWORKING && echo return
```
because now the code looks more like what is meant to happen
```
When DHCLIENT shoud be used and STATIC_NETWORKING should not be used, then 'return'.
```

Reply:
Hmpf!
That boolean variables like
```
# Say "y", "Yes" (or any not empty string) to enable static networking (overrules USE_DHCLIENT):
USE_STATIC_NETWORKING=
```
are a pain because of the "or any not empty string" which means
```
USE_STATIC_NETWORKING=false
```
means 'true' which so obviously contradics what one expects
that I think no user uses that in his config file so that we can
safely remove the "or any not empty string" part in default.conf like
```
# Say "y", "Yes", etc, to enable static networking (overrules USE_DHCLIENT):
USE_STATIC_NETWORKING=
```
without causing any real regression for any real user.